src/effects/Fire.js

```diff
@@ -129,6 +137,33 @@ export default class Effect {
       // Initialize the fire-preset dropdown.
       Effect._createFirePresets(dialog);
     }
+
+    // enables and disables the color buttons
+    function EnableDisableColorButtons(dialog, state) {
```

Comment:
Just for consistency.

```suggestion
    function enableDisableColorButtons(dialog, state) {
```